horizon/opal_relay_api.py

```diff
@@ -129,17 +124,14 @@ async def relay_session(self) -> ClientSession:
                 except TypeError:
                     try:
                         text = await response.text()
-                    except:
-                        text = None
-                    raise RelayAPIException(
-                        "relay-jwt-api",
-                        response.status,
-                        f"Server responded to token request with an invalid result: {text}",
-                    )
+                    except Exception as e:
+                        raise RelayAPIError(
+                            "relay-jwt-api",
+                            response.status,
+                            f"Server responded to token request with an invalid result: {text}",
+                        ) from e
```

Comment:
this isn't exactly the same behavior, before the change it always raised, not it doesn't always raise

Reply:
Nice catch!! we missed it last PR
https://github.com/permitio/PDP/pull/226/files#diff-7e22143ec950747331a9f47bdeb54523cf86ca238c53b909d5ce5fdbb930b360R127-R132

Fixing it now

Reply:
Fixed